crates/solidity/outputs/rust/lib/src/generated/kinds.rs

```diff
@@ -296,10 +297,10 @@ pub enum TokenKind {
     ZeroX,
 }
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
 pub enum RuleKind {
+    ABICoderPragma,
```

Comment:
I wonder what caused that capitalization :O

Reply:
I have no idea. I want to revisit this issue to audit naming.